backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/data/DataExportAudit_StepDef.java

```diff
@@ -311,4 +311,18 @@ public void change_bpartner_location(@NonNull final DataTable dataTable)
 
 		InterfaceWrapperHelper.save(bPartnerLocation);
 	}
+
+	@And("the following c_bpartner is changed")
+	public void change_bpartner(@NonNull final DataTable dataTable)
```

Comment:
AFAIU this stepdef
* doesn't require a table, because it about one row and one column.
* doesn't belong into DataExportAudit_StepDef, but rather into C_BPartner_StepDef

LE: feel free to keep it a table if you want, but then please, iterate over all rows, and add a 2nd column for `Name2`, so it can be extended as needed later